### PR TITLE
Use `java` directly for starting appications.

### PR DESCRIPTION
**Pull request by Amygos**
Switch to `java` as default method for start tomcat application container,
applications that still need `jsvc`, can use the systemd `tomcat8-jsvc@.service`
unit template file.

NethServer/dev#5788

### CODE REVIEW

File: tomcat8-jsvc@.service [hunk 1]
```diff
@@ -1,29 +1,31 @@
 # Systemd unit file for tomcat instances.
 #
 # To create clones of this service:
-# 0. systemctl enable tomcat8-nojsvc@name.service
+# 0. systemctl enable tomcat8@name.service
```
Comment (gsanchietti) [inline tomcat8-jsvc@.service]: I think in the comment, `tomcat8@name.service` should be renamed to `tomcat8-nojsvc@name.service.

File: tomcat8@.service [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Systemd unit file for tomcat instances.
-# 
+#
 # To create clones of this service:
-# 0. systemctl enable tomcat8@name.service
+# 0. systemctl enable tomcat8-nojsvc@name.service
```
Comment (gsanchietti) [inline tomcat8@.service]: I think in the comment, `tomcat8-nojsvc@name.service` should be renamed to `tomcat8@name.service.

**Review by DavidePrincipi — APPROVED** [review]
Exit status 143?

**Comment by Amygos** [comment]
> Exit status 143?

A Tomcat java application when is stopped with a `SIGTERM`, return a exit code `143` for inform you that it was terminate by a `SIGTERM`, the `SuccessExitStatus=143` directive tell to systemd to accept `143` as a success exit status, otherwise will mark the unit as failed after the stop.

**Pull request merged**